code/drv_bms/src/wattrex_driver_bms/drv_bms.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/python3
+'''
+Driver of bms.
+'''
+
+#######################        MANDATORY IMPORTS         #######################
+from __future__ import annotations
+from sys import path
+import os
+
+#######################         GENERIC IMPORTS          #######################
+from enum import Enum
+#######################       THIRD PARTY IMPORTS        #######################
+
+#######################      SYSTEM ABSTRACTION IMPORTS  #######################
+path.append(os.getcwd())
+from system_logger_tool import sys_log_logger_get_module_logger # pylint: disable=wrong-import-position
+log = sys_log_logger_get_module_logger(__name__)
+from system_shared_tool import SysShdIpcChanC # pylint: disable=wrong-import-position
+
+#######################          PROJECT IMPORTS         #######################
+from can_sniffer import DrvCanCmdTypeE, DrvCanCmdDataC, DrvCanFilterC, DrvCanMessageC
+from wattrex_driver_base import DrvBaseStatusE, DrvBaseStatusC
+
+#######################          MODULE IMPORTS          #######################
+
+#######################              ENUMS               #######################
+_MAX_MSG = 100
```

Comment:
Move to any common file in order to centralize data used to manage time, and chans names and size